src/components/RenderRouter/HomeChurchItem.scss

```diff
@@ -270,7 +277,7 @@
       }
       .HomeChurchItemListData {
             overflow-y: auto;
-            height: 80vh;
+            height: 70vh;
```

Comment:
This height needs to reduced more since the scrolling area is spilling onto the footer. I'd suggest using `calc()` to determine how much empty vertical space you have.

![Screenshot_2020-10-31 Find a Home Church](https://user-images.githubusercontent.com/48423418/97792493-5be99f80-1bb5-11eb-832d-4eb73a6ecb88.png)

Screen resolution: 1280x550 (laptop).